l10n_br_currency_rate_update/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
License missing

Reply:
@alexey-pelykh 
Thank you very much for the review. We however don't put license headers anymore in these `__init__.py` almost empty files at the OCA to avoid bloating the code.

Reply:
There's a template https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
oh you are right indeed. I mean we don't put the copyrights in `__init__.py `anymore but indeed the single line license header is usually set.

Reply:
Done!